lib/runtime/evaluation.coffee

```diff
@@ -28,6 +26,12 @@ module.exports =
           @showError r, result.highlights
         notifications.show "Evaluation Finished"
 
+  evalCode: (text, line=1) ->
```

Comment:
Is there a particular reason for sending this information, or is this here just so that you can evaluate a code string? It might make more sense to implement a `workspace` command and call that via the `rpc()` – see for example the `exit` command.